dftimewolf/lib/logging_utils.py

```diff
@@ -0,0 +1,71 @@
+"""Module providing custom logging formatters."""
+import logging
+import random
+
+def _GenerateColorSequences():
+  """Generates ANSI codes for 256 colors."""
+  sequences = []
+  for i in range(0, 16):
+    for j in range(0, 16):
+      code = str(i * 16 + j)
+      seq = '\u001b[38;5;' + code + 'm'
+      sequences.append(seq)
+  return sequences
+
+COLOR_SEQS = _GenerateColorSequences()
+RESET_SEQ = '\u001b[0m'
+
+# Cherrypick a few interesting values. We still want the whole list of colors
+# so that modules have a good amount colors to chose from.
+# pylint: disable=unbalanced-tuple-unpacking
+BLACK, RED, GREEN, YELLOW, BLUE, MAGENTA, CYAN, WHITE = COLOR_SEQS[8:16]
+BG_RED = '\u001b[41m'  # Red background
+BOLD = '\u001b[1m'  # Bold / bright modifier
+
+LOG_FORMAT = ('[%(asctime)s] [{0:s}{color:s}%(name)-20s{1:s}] %(levelname)-8s'
+              ' %(message)s')
```

Comment:
what about `%(name)s/%(levelname)s` instead of just `%levelname`....

that is the `name` gives you the name of the logger. In the case of timesketch api/importer, etc thsi would give you where this is coming from (since you already `killed` the root logger, you should only get back records from a named logger)

Reply:
we already have `(name)s` in brackets. This is what it looks like:

```
[2020-07-09 18:06:05,187] [TimesketchExporter  ] INFO     Sketch 23 created
```

`TimesketchExporter` being the name of the logger (named after the class name of the module that initializes it; this can be changed)

Reply:
some how I missed the name there... sorry about that ;)